web/src/utils/transformDataHelper.tsx

```diff
@@ -94,10 +94,54 @@ export function getProfileContacts(contactSet: any[]): object {
   contactSet.forEach((contact: any) => {
     if (contact.roleId === 1) {
       contacts.POEmail = contact.email;
+      contacts.POName = contact.firstName + ' ' + contact.lastName;
+      contacts.POGithubId = contact.githubId;
     }
     if (contact.roleId === 2) {
       contacts.TCEmail = contact.email;
+      contacts.TCName = contact.firstName + ' ' + contact.lastName;
+      contacts.TCGithubId = contact.githubId;
     }
   });
   return contacts;
-};
\ No newline at end of file
+};
+
+// convert datetime string from YYYY-MM-DDTHH:MM:SSZ to DD-MM-YYYY HH:MM
+function convertDatetime(ISODatetimeString: string): string {
+  const splitted = ISODatetimeString.split('T');
+  const HHMM = splitted[1].replace(/\..+/, '').split(':');
+  HHMM.pop();
+  return splitted[0].split('-').reverse().join('-') + ' ' + HHMM.join(':');
+}
+
+
+export function transformJsonToCsv(objArray: any) {
+  const array = typeof objArray !== 'object' ? JSON.parse(objArray) : objArray;
+
+  array.forEach((item: any) => {
+    item.createdAt = convertDatetime(item.createdAt);
+    item.updatedAt = convertDatetime(item.updatedAt);
+  });
+
+  let str = '';
+  let line = '';
+
+  for (let index in array[0]) {
+    line += index + ',';
+  }
+
+  line = line.slice(0, -1);
+  str += line + '\r\n';
+
+  for (let i = 0; i < array.length; i++) {
+    line = '';
+
+    for (let index in array[i]) {
+      line += array[i][index] + ',';
```

Comment:
As discussed, descriptions that contained commas were being mistaken as separators. The following resolved this issue with my local tests. By encapsulating the entries in escaped "" it protects internal commas.
```suggestion
      line += '\"' + array[i][index] + '\"' + ',';
```